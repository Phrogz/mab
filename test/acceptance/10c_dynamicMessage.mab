p( "Test#10c" )
setSlot( "myMessage", Message new )
myMessage setSlot( "identifier", "p" )
myMessage addArgument( "Hello, world!" )

setSlot( "myExpr", Expression new )
myExpr appendMessage( myMessage )

myExpr eval