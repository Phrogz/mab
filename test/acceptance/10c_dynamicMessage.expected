Test#10c
Hello, world!